### ISSUE TITLE
Unclosed file warning in test_image.test_imread_pil_uint16

### ISSUE THREAD

**Issue by mdboom** [matplotlib/matplotlib#5364]
```
matplotlib.tests.test_image.test_imread_pil_uint16 ... /home/mdboom/.pyenv/versions/miniconda3-3.16.0/lib/python3.4/site-packages/PIL/Image.py:665: ResourceWarning: unclosed file <_io.TextIOWrapper name='/matplotlib/tests/baseline_images/test_image/uint16.tif' mode='r+' encoding='UTF-8'>
  self.load()
/matplotlib/image.py:1304: ResourceWarning: unclosed file <_io.BufferedReader name='/matplotlib/tests/baseline_images/test_image/uint16.tif'>
  im = pilread(fname)
ok
```

**Comment by tacaswell** [comment]
I think both @jenshnielsen and I took a pass at this and concluded the issue was in PIL.

**Comment by jenshnielsen** [comment]
Yes I tried fixing it in #4619 but that solution breaks other stuff

**Comment by mdboom** [comment]
Ok.  I have no problem punting -- it's helpful to have this issue exist to make it easily searchable, though.

**Comment by anntzer** [comment]
Was fixed by #5515.
